File: PraktikumsverwaltungWebService/src/java/pkgData/Teacher.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package pkgData;

import org.bson.types.ObjectId;

/**
 *
 * @author schueler
 */
public class Teacher extends Person {
    private boolean isAdmin;

    public Teacher() {
    }

    public Teacher(String id, String username, String password, String firstName, String lastname, String email, boolean isAdmin, boolean isActive) {
        super(id, username, password, firstName, lastname, email, isActive);
        this.isAdmin = isAdmin;
    }

    public boolean isIsAdmin() {
        return isAdmin;
    }

    public void setIsAdmin(boolean isAdmin) {
        this.isAdmin = isAdmin;
    }
    
}

File: DesktopApp/Praktikumsverwaltung_DesktopApp/Praktikumsverwaltung_DesktopApp/pkgData/Teacher.cs
using MongoDB.Bson.Serialization.Attributes;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Praktikumsverwaltung_DesktopApp.pkgData
{
    class Teacher : Person
    {
        public bool IsAdmin { get; set; }

        public Teacher(string id, string firstname, string lastname, string email, string username, string password, bool isActive, bool isAdmin)
        : base(id, firstname, lastname, email, username, password, isActive)
        {
            IsAdmin = isAdmin;
        }
    }
}
